lib/hammer_cli_foreman/commands.rb

```diff
@@ -202,6 +202,9 @@ def send_request
         },
         e.resource
       )
+    rescue Exception => e
+      HammerCLIForeman.foreman_api_connection.api.clear_credentials if e.to_s.include?"401"
```

Comment:
Or even - would it make sense to catch the error in apipie-bindings while we do not touch any other Hammer objects?